README.md

```diff
@@ -36,7 +36,7 @@ Add following lines to `.bashrc` or `.zshrc`:
 BASE16_SHELL_PATH="$HOME/.config/base16-shell"
 [ -n "$PS1" ] && \
   [ -s "$BASE16_SHELL_PATH/profile_helper.sh" ] && \
-    source "$BASE16_SHELL_PATH/profile_helper.sh"
+    "$BASE16_SHELL_PATH/profile_helper.sh"
```

Comment:
@JamyGolden this is probably the sticking point...
if people _don't_ change this from `source` to just executing the profile helper in their RC file, the `exit` will exit their shell!

Reply:
the alternative is something like this, which works if sourced or if executed:
```shell
# If $BASE16_THEME is set, this has already been loaded. This guards
# against a bug where this script is sourced two or more times.
if [ -z "$BASE16_THEME" ]; then

  # Load the active theme
  if [ -e "$BASE16_SHELL_COLORSCHEME_PATH" ]; then
    # Get the active theme name from the export variable in the script
    current_theme_name=$(grep 'export BASE16_THEME' "$BASE16_SHELL_COLORSCHEME_PATH")
    current_theme_name=${current_theme_name#*=}
    set_theme "$current_theme_name"
  # If a colorscheme file doesn't exist and BASE16_THEME_DEFAULT is set,
  # then create the colorscheme file based on the BASE16_THEME_DEFAULT
  # scheme name
  elif [ -n "$BASE16_THEME_DEFAULT" ]; then
    set_theme "$BASE16_THEME_DEFAULT"
  fi
fi
```